test/dominatingset/minimal_dom_set.jl

```diff
@@ -2,45 +2,45 @@
  
     g0 = SimpleGraph(0)
     for g in testgraphs(g0)
-        d = @inferred(dominating_set(g, MinimalDominatingSet(); seed=3))
+        d = @inferred(dominating_set(g, MinimalDominatingSet(); rng=RNG, seed=3))
```

Comment:
```suggestion
        d = @inferred(dominating_set(g, MinimalDominatingSet(); rng=RNG))
```
there does not seem to be any point of specifying a seed, when we also specify `rng`.

Reply:
It seeds the `rng` with this specific value, so it does, even if admittedly, we don't test the numbers stream per se.